index.html

```diff
@@ -218,7 +218,7 @@ <h2 class="about-me-title">About Me</h2>
             </section>
             <section class="footnote">
                 <div class="brief-note-div">
-                    <h3 class="brief-note">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+                    <h3 class="brief-note">I'm always <br> interested in hearing about new projects, so if you'd like to <br> chat please get in touch.</h3>
                 </div>
                 <form action="https://formspree.io/f/meqdypvw" method="post" class="form-container">
```

Comment:
- [ ] [OPTIONAL] I think it would be better if you match the text of your button to the Figma template. It would make more sense if the button showed `Get in touch` instead of `Get my resume` since the user visiting the page is most probably trying to send you a message. 🎃

**Your Design**
![image](https://user-images.githubusercontent.com/17243325/207444254-7642c9a5-dce3-442a-97c4-979bf8e1c012.png)

**Figma Design**
![image](https://user-images.githubusercontent.com/17243325/207444400-9fa23a72-f473-4dc9-a484-8d2667cf3596.png)